### DISCUSSION TITLE
Backlight dimmer  - DSB - Additional Memory lists

### DISCUSSION THREAD

**Discussion by iu0azb** [egzumer/uv-k5-firmware-custom#195] [Ideas]
Hello egzumer,
first of all, want to thank you to have made the best firmware for this Radio, really the best after have tried an handful of versions.  Your is the most complete and functional :) 
After days of use I would just have few wonders.. 
- Dimm the backlight but I'm not sure if this can control in pwm trough software, unfortunately I've found no schematic related.
- TX in DSB on ham bands ( I successfully made a several VHF contatcs with the IJV firmware that is really good on this )
- Think to implement more than 2 list of memory set

Thanks a lot IU0AZB

**Comment by egzumer** [comment]
`BLMin` - minimal backlight brightness, when the screen backlight turns OFF it will go dim to this value
`BLMax` - maximal backlight brightness, when the screen backlight turns ON it will turn bright to this value

Not doing TX in DSB, this is an FM transceiver for VHF/UHF.

Memory list is original QS thing, one of a base functionalities of the radio, changing this would be a big compatibility issue, I'm not touching that.

**Reply by iu0azb** [reply]
Thank you for your feedbacks.. my fault to have not look better to find BLMax before note to you.
